pyproject.toml

```diff
@@ -409,6 +400,11 @@ fixture-parentheses = false
 combine-as-imports = true
 required-imports = ["from __future__ import annotations"]
 
+[tool.ruff.format]
+line-ending = 'lf'
+skip-magic-trailing-comma = true
+preview = true
```

Comment:
holding off from merging this until https://github.com/astral-sh/ruff/issues/6936 is doned